src/languages/bs.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Bosnian (bosanski) language functions
  */
-import BananaLanguage from './language/index.js'
```

Comment:
We're now loading `BananaLanguage` from `language.js`, can we remove the export from `index.js`?

Reply:
The emitter.js is importing all languages.
`import languages from './languages/index.js'`
So we need all exports from languages/index.js